pkgs/applications/networking/instant-messengers/wire-desktop/default.nix

```
@@ -15,9 +16,10 @@ let
     freetype
     gdk_pixbuf
     glib
-    gnome2.GConf
-    gtk2
+    gtk3
+    at_spi2_atk
```

```suggestion
    at-spi2-atk
```
That attribute is an old alias.